src/util/jira.ts

```diff
@@ -6,21 +6,33 @@ const base = 'https://mattermost.atlassian.net';
 
 export class JiraClient {
   private token: string | undefined;
+  private email: string | undefined;
 
-  constructor(token?: string) {
+  constructor(token?: string, email?: string) {
     if (token) {
       this.token = token;
     } else {
       console.warn(
         'no token provided, fetching from Jira as an anonymous user',
       );
     }
+    if (email) {
+      this.email = email;
+    } else {
+      console.warn(
+        'no email provided. Required email to fetch components from Jira',
+      );
+    }
   }
 
   private _auth(r: Request): Request {
     const req = new Request(r);
     if (this.token) {
-      req.headers.set('Authorization', `Bearer ${this.token}`);
+      const encoder = new TextEncoder();
+      const data = encoder.encode(`${this.email}:${this.token}`);
+      const base64 = btoa(String.fromCharCode(...new Uint8Array(data)));
+      const authString = `Basic ${base64}`;
+      req.headers.set('Authorization', authString);
```

Comment:
nit: I'd request for the `Basic` to go directly when setting up the authorization and not through a separate variable.

```suggestion

      req.headers.set('Authorization', `Basic ${base64}`);
```

Reply:
thank you @saturninoabril , I've made the change